arrow-libs/core/arrow-core/api/jvm/arrow-core.api

```diff
@@ -1013,12 +1044,13 @@ public class arrow/core/raise/RaiseAccumulate : arrow/core/raise/Raise {
 	public final fun mapValuesOrAccumulate (Ljava/util/Map;Lkotlin/jvm/functions/Function2;)Ljava/util/Map;
 	public fun raise (Ljava/lang/Object;)Ljava/lang/Void;
 	public final fun raiseErrors ()Ljava/lang/Void;
+	public final fun recover (Lkotlin/jvm/functions/Function1;Lkotlin/jvm/functions/Function1;)Ljava/lang/Object;
 	public final fun withNel (Lkotlin/jvm/functions/Function1;)Ljava/lang/Object;
 }
 
 public final class arrow/core/raise/RaiseAccumulate$Error : arrow/core/raise/RaiseAccumulate$Value {
-	public fun <init> (Larrow/core/raise/RaiseAccumulate;)V
-	public final fun getRaise ()Larrow/core/raise/RaiseAccumulate;
```

Comment:
This was an accidental exposure in the context parameters PR. It didn't get published though, so this removal here is safe.